QualityControl/public/common/filters/model/FilterModel.js

```diff
@@ -31,6 +31,7 @@ export default class FilterModel extends Observable {
     this.model = model;
     this.filterService = new FilterService(this);
     this._filterMap = {};
+    this.visible = true;
```

Comment:
Normally, for boolean values the convention for naming is something like `is/has`
`isVisible` - in this cse